source/adminguide/storage.rst

```diff
@@ -183,6 +183,28 @@ secondary storage. For information about how to install and configure
 secondary storage through the CloudStack UI, see the Advanced
 Installation Guide. about-secondary-storage>`_
 
+Migration of data between secondary storages is now supported. One may choose
+to completely migrate the data or migrate data such that the stores
+are balanced by choosing the appropriate Migration Policy. In order to facilitate
+distributing the migration load, SSVMs are spawned up if a file transfer takes
+more that a defined threshold. Following are the Global setting values to one may
```

Comment:
`takes more that a defined threshold` :arrow_right: `takes more than a defined threshold`